### PR TITLE
fix: convert paths to relative for vitePreprocess sourcemap sources

### PR DESCRIPTION
**Pull request by dominikg**
improves #569  at least for directories with spaces outside of workspace root.

prevents duplicate sources entries, see updated test snapshot

### CODE REVIEW

File: packages/vite-plugin-svelte/src/preprocess.ts [hunk 1]
```diff
@@ -7,6 +6,8 @@ import type { Preprocessor, PreprocessorGroup } from 'svelte/types/compiler/prep
 const supportedStyleLangs = ['css', 'less', 'sass', 'scss', 'styl', 'stylus', 'postcss', 'sss'];
 const supportedScriptLangs = ['ts'];
 
+import { mapSourcesToRelative } from './utils/sourcemaps';
```
Comment (bluwy) [inline packages/vite-plugin-svelte/src/preprocess.ts:9]: Would be nice to move this up together with the other imports.

**Review by bluwy — COMMENTED** [review]
one small nit but this looks like a nice change.

File: packages/vite-plugin-svelte/src/preprocess.ts (hunk 1 above)
Comment (dominikg) [reply packages/vite-plugin-svelte/src/preprocess.ts]: done

File: .changeset/eight-keys-hug.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/vite-plugin-svelte': patch
+---
+
+fix(vitePreprocess): use relative paths in sourcemap sources"
```
Comment (bluwy) [inline .changeset/eight-keys-hug.md:5]:
```suggestion
fix(vitePreprocess): use relative paths in sourcemap sources
```

**Review by bluwy — APPROVED** [review]

**Pull request merged**